packages/core/src/types/asyncapi2.ts

```diff
@@ -1099,12 +1086,32 @@ export const AsyncApi2Types: Record<string, NodeType> = {
 
   ServerBindings,
   ChannelBindings,
+  MessageBindings,
+  OperationBindings,
+};
+
+export const AsyncApi2Types: Record<string, NodeType> = {
+  ...AsyncApi2Bindings,
+  Root,
+  Tag,
+  TagList: listOf('Tag'),
```

Comment:
should this remain consistent with other `type` files.  `NamedTags: listOf('Tag')` ??

Reply:
same. please check answer below)